recipes/itk-dev/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "4.11.0" %}
+
+package:
+  name: itk-dev
+  version: "{{version}}"
+
+source:
+  git_url: git://itk.org/ITK.git
+  git_tag:  v{{version}}
```

Comment:
We generally prefer using a tarball to a git clone. Is there a tarball or archive that we could use?